tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist =
+    {py27,py34,py35}-django{18,19,110}-{sqlite,postgres,mysql}
```

Comment:
Add `flake8` as another env.

A good reference tox.ini that works well is:

https://github.com/etianen/django-reversion/blob/master/tox.ini

It's pretty much identical to how you've done things. :)